### ISSUE TITLE
Wrong restructuring of height in class and element selectors

### ISSUE THREAD

**Issue by accetone** [css/csso#411]
Please check following input and output. Class `layout` will have `height: 60px` which is incorrect.

Input
```css
html {
  width: 100%;
  height: 100%;
  margin: 0;
}

.header {
  width: 100%;
  height: 60px;
}

.layout {
  width: 100%;
  height: 100%;
  margin: 0;
  padding: 0;
}
```

Output
```css
.layout,
html {
  height: 100%;
  margin: 0
}

html {
  width: 100%
}

.header,
.layout {
  width: 100%;
  height: 60px
}

.layout {
  padding: 0
}
```

**Comment by charlessuh** [comment]
@accetone I've taken a [stab at a fix](https://github.com/css/csso/compare/master...charlessuh:bimerge?expand=1) as a user myself. Does the following look correct before I open a PR?

```
html {
    width: 100%
}

.header,.layout {
    width: 100%;
    height: 60px
}

.layout,html {
    height: 100%;
    margin: 0
}

.layout {
    padding: 0
}
```

**Comment by accetone** [comment]
@charlessuh thanks, your output look correct.

**Comment by lahmatiy** [comment]
Fix published in [4.0.3](https://github.com/css/csso/releases/tag/v4.0.3)
